pkg/anonymization/anonymizer.go

```diff
@@ -487,9 +487,11 @@ func (anonymizer *Anonymizer) IsObfuscationEnabled() bool {
 	if obfuscateNetworking(obfuscation) {
 		return true
 	}
+
 	if anonymizer.dataPolicy != "" {
```

Comment:
This field is now a collection, maybe it'd be a good idea to refactor this conditional and the return statement.